oxidation/libparsec/crates/testbench/src/coolorg.rs

```diff
@@ -0,0 +1,161 @@
+    ));
+    users.push(TestbenchUserData::new(
+        alice1_id.user_id().clone(),
+        Some(HumanHandle::new("alice@example.com", "Alicey McAliceFace").unwrap()),
+        PrivateKey::from(hex!(
+            "74e860967fd90d063ebd64fb1ba6824c4c010099dd37508b7f2875a5db2ef8c9"
+        )),
+        UserProfile::Admin,
+        EntryID::from_hex("a4031e8bcdd84df8ae12bd3d05e6e20f").unwrap(),
+        SecretKey::from(hex!(
+            "26bf35a98c1e54e90215e154af92a1af2d1142cdd0dba25b990426b0b30b0f9a"
+        )),
+        CertificateSignerOwned::Root,
+        &root_signkey,
+        dt,
+    ));
+    devices.push(TestbenchDeviceData::new(
+        alice1_id.clone(),
+        Some("My dev1 machine".parse().unwrap()),
+        alice1_signkey.clone(),
+        SecretKey::from(hex!(
+            "125a78618995e2e0f9a19bc8617083c809c03deb5457d5b82df5bcaec9966cd4"
+        )),
+        CertificateSignerOwned::Root,
+        &root_signkey,
+        dt,
+    ));
+
+    // 2000-01-02: alice@dev1 creates bob@dev1 (standard profile)
+
+    let dt: DateTime = "2000-01-02T00:00:00Z".parse().unwrap();
+    let bob1_id: DeviceID = "bob@dev1".parse().unwrap();
+    let bob1_signkey = SigningKey::from(hex!(
+        "85f47472a2c0f30f01b769617db248f3ec8d96a490602a9262f95e9e43432b30"
+    ));
+    users.push(TestbenchUserData::new(
+        bob1_id.user_id().clone(),
+        Some(HumanHandle::new("bob@example.com", "Boby McBobFace").unwrap()),
+        PrivateKey::from(hex!(
+            "16767ec446f2611f971c36f19c2dc11614d853475ac395d6c1d70ba46d07dd49"
+        )),
+        UserProfile::Standard,
+        EntryID::from_hex("71568d41afcb4e2380b3d164ace4fb85").unwrap(),
+        SecretKey::from(hex!(
+            "65de53d2c6cd965aa53a1ba5cc7e54b331419e6103466121996fa99a97197a48"
+        )),
+        CertificateSignerOwned::User(alice1_id.clone()),
+        &alice1_signkey,
+        dt,
+    ));
+    devices.push(TestbenchDeviceData::new(
+        bob1_id.clone(),
+        Some("My dev1 machine".parse().unwrap()),
+        bob1_signkey.clone(),
+        SecretKey::from(hex!(
+            "93f25b18491016f20b10dcf4eb7986716d914653d6ab4e778701c13435e6bdf0"
+        )),
+        CertificateSignerOwned::User(alice1_id.clone()),
+        &alice1_signkey,
+        dt,
+    ));
+
+    // 2000-01-03: alice@dev1 creates alice@dev2
+
+    let dt: DateTime = "2000-01-03T00:00:00Z".parse().unwrap();
+    let alice2_id: DeviceID = "alice@dev2".parse().unwrap();
+    devices.push(TestbenchDeviceData::new(
+        alice2_id,
+        Some("My dev2 machine".parse().unwrap()),
+        SigningKey::from(hex!(
+            "571d726cc5586b4bfd5b20d8af2365cf8bb8c881b4925794e6e38cdcc5ec82ef"
+        )),
+        SecretKey::from(hex!(
+            "fd64fba009dc635af6662f45753b8c9772ff6e214e203e9d61ce029550e250f7"
+        )),
+        CertificateSignerOwned::User(alice1_id.clone()),
+        &alice1_signkey,
+        dt,
+    ));
```

Comment:
We could create a macro for that :thinking:

Reply:
better do that in another PR (typically when we will add further stuff to the template like manifests) to have this one merged asap